extensions/datastores/hbase/coprocessors/src/main/java/org/locationtech/geowave/datastore/hbase/coprocessors/HBaseBulkDeleteEndpoint.java

```diff
@@ -326,4 +335,36 @@ public void start(final CoprocessorEnvironment env) throws IOException {
   public void stop(final CoprocessorEnvironment env) throws IOException {
     // nothing to do
   }
+
+  public static void main(final String[] args) throws Exception {
```

Comment:
This looks like left over test code that should probably be removed.

Reply:
good catch